src/hotspot/cpu/x86/sharedRuntime_x86_64.cpp

```diff
@@ -3247,7 +3372,7 @@ RuntimeStub* SharedRuntime::generate_resolve_blob(address destination, const cha
   // allocate space for the code
   ResourceMark rm;
 
-  CodeBuffer buffer(name, 1200, 512);
+  CodeBuffer buffer(name, 1752, 512);
```

Comment:
What cause the need to such increase of size?

Reply:
Stub makes multiple save restoration calls, additional buffer size needed to accommodate to move EGPRs  from / to stack. Further constraining the size.